misc/jenkins/compile_other_versions/prepare_bundle.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
 SCRIPT_NAME="prepare_bundle.sh"
-echo "Entering $SCRIPT_NAME with $BUNDLE_NAME"
+echo "Entering $SCRIPT_NAME with ${1}"
```

Comment:
shall we 
``BUNDLE_NAME=${1}``
on top?

Reply:
Good idea.